### ISSUE TITLE
Create Abstract of our project

### ISSUE THREAD

**Issue by christianLeomil** [aateg/tatu-re#49]

**Comment by christianLeomil** [comment]
Send Abstract until 31.01, send also group name when abstract is finished

**Comment by christianLeomil** [comment]
The stock market has seen a long bullish period in the past decade, with various speculation tactics bringing wealth to practitioners. However, this year has proven to be a difficult time for predicting stock market trends. In this project, we aimed to address this challenge by building a recommender engine for the SPDR S&P 500 ETF. Additionally, we also tested our model on other stocks such as DAX and Nikkei to test its generalizability.

Our goal was to provide daily recommendations to users on whether they should buy, hold, or sell the ETF, with the ultimate aim of maximizing virtual earnings over a 2-week interval. To accomplish this, we employed machine learning models, such as Linear Regression and Hidden Markov Model to analyze stock market data and make predictions.

The best results were achieved using a Hidden Markov Model, which was confirmed through statistical techniques such as Welch unpooled test and Wilcoxon signed-rank test. These tests were performed using the distributions of the area under curve for the stock market curve and the designed strategy curve.

Finally, our designed routine sent daily emails to the users before the stock market opens, which included links to a dashboard displaying more information about stock performance in comparison to the developed strategy performance.

Overall, this project provided an opportunity to gain hands-on experience in using machine learning techniques to tackle a real-world problem, and to learn about the various considerations and steps involved in building a recommender engine. The project allowed us to gain an understanding of the importance of data preparation, iterative design, and model serving, as well as the process of model selection and comparison.
